AnnoyancesFilter/sections/annoyances.txt

```diff
@@ -1711,6 +1711,7 @@ perfect-english-grammar.com##form[action^="https://app.convertkit.com/"]
 businessinsider.de##.bi-newsletter
 nubank.com.br##.newsletter-box-widget
 nubank.com.br###newsletter-experiment
+||byjus.com/byjusweb/js/tracking-2.min.js
```

Comment:
`||byjus.com/byjusweb/js/tracking-2.min.js` to spyware filter
`||livechatinc.com^$domain=byjus.com` to annoyances